Dockerfile.test.gpu

```diff
@@ -116,7 +116,7 @@ RUN pip install "Pillow<7.0" --no-deps
 
 # Install MXNet.
 RUN if [[ ${MXNET_PACKAGE} == "mxnet-nightly" ]]; then \
-        pip install --pre mxnet-cu101mkl -f https://dist.mxnet.io/python/all; \
+        pip install --pre mxnet-cu101 -f https://dist.mxnet.io/python/all; \
```

Comment:
this is discontinued as NVIDIA only supports the latest two major versions and minor versions. can the horovod project update its dependencies?